crates/nu-protocol/src/config/mod.rs

```diff
@@ -627,6 +635,22 @@ impl Value {
                     "highlight_resolved_externals" => {
                         process_bool_config(value, &mut errors, &mut config.highlight_resolved_externals);
                     }
+                    "plugins" => {
```

Comment:
We can add the same key to our `sample_config/default_config.nu`, with an empty record.  So user can detect such feature

Reply:
I added this with a link to the nushell plugins page in ab9b6abaaf0602a76525ee919f2f4829d4bd55f9

When this is approved I can write the new section for describing plugin configuration